src/gpu_fdinfo.h

```diff
@@ -67,16 +67,28 @@ class GPU_fdinfo {
         : module(module)
         , pci_dev(pci_dev)
     {
-        SPDLOG_DEBUG("GPU driver is \"{}\"", module);
+        SPDLOG_INFO("GPU driver is \"{}\"", module);
```

Comment:
We don't want to print things every time unless it's not expected, this should be debug

Reply:
Gpu driver is also printen only once

Reply:
As mentioned earlier, the problem is that nothing is mentioned once when launched from steam etc, it will spam the logs